Cargo.lock

```diff
@@ -42,7 +42,7 @@ dependencies = [
  "atspi-common",
  "serde",
  "thiserror",
- "zvariant",
+ "zvariant 4.2.0",
```

Comment:
Is it possible to use a single version for this crate?

Reply:
Doesn't look like it

Reply:
Actually, yes.